src/sage/geometry/hyperplane_arrangement/ordered_arrangement.py

```diff
@@ -0,0 +1,650 @@
+r"""
+Ordered Hyperplane Arrangements
+
+The :class:`HyperplaneArrangements` ordered the arrangements in a hyperplane
+independently of the way the hyperplanes are introduced. With the class
+:class:`OrderedHyperplaneArrangements` it is allowed to fix an order by
+the user, what is needed for some properties, e.g., topological one.
+Besides this fact, it inherits the rest of the properties from :class:`HyperplaneArrangements.`
```

Comment:
```suggestion
The :class:`HyperplaneArrangements` orders the hyperplanes in a arrangement
independently of the way the hyperplanes are introduced. The class
:class:`OrderedHyperplaneArrangements` fixes an order specified by
the user. This can be needed for certain properties, e.g., ***.
There are no other differences with usual hyperplane arrangements.
```
where `***` should have a specific property.

Reply:
Done with specific properties.